easybuild/easyconfigs/i/InParanoid/InParanoid-5.0.79d8e39-GCC-10.3.0.eb

```diff
@@ -0,0 +1,65 @@
+##
+# This is a contribution from SIB Swiss Institute of Bioinformatics
+# Homepage:     https://www.sib.swiss/research-infrastructure/competence-centers/vital-it
+#
+# Authors::     Sebastien Moretti <sebastien.moretti@sib.swiss>
+#
+##
+
+easyblock = 'Tarball'
+
+name = 'InParanoid'
+version = '5.0.79d8e39'
+
+homepage = 'https://inparanoid.sbc.su.se'
+description = """InParanoid: ortholog groups with inparalogs."""
+software_license = 'LicenseGPLv3'
+
+toolchain = {'name': 'GCC', 'version': '10.3.0'}
+
+source_urls = ['https://bitbucket.org/sonnhammergroup/inparanoid/get/']
+sources = ['79d8e39bb243d403ce57b699c1ae104a8a640389.tar.gz']
+checksums = ['efe1a35bbb70608385780c4bf62969a29c46dd8132fbe472034422d23ad4b0c2']  # for commit 79d8e39
+
+builddependencies = [('binutils', '2.36.1')]
+
+# NOTE some dependencies are not listed on the web site such as Java!
+dependencies = [
+    ('Perl',    '5.32.1'),  # brings libmoose-perl and libparallel-forkmanager-perl
+    ('BioPerl', '1.7.8'),
+    ('Java',    '11',     '',              SYSTEM),
+    ('BLAST',   '2.2.26', '-Linux_x86_64', SYSTEM),
+    ('DIAMOND', '2.0.13'),
+]
+
+# NOTE inparanoid.pl uses only local (same directory) paths, so patch it
+postinstallcmds = ["rm -rf %(installdir)s/docker/ %(installdir)s/singularity/ && "
+                   "chmod a+rx %(installdir)s/*.pl && "
+                   "chmod -x %(installdir)s/*.jar && "
+                   "perl -i -pe 's|helpfile = \"help\"|helpfile = \"%(installdir)s/help\"|' "
+                   "%(installdir)s/inparanoid.pl && "
+                   "perl -i -pe 's|seqstat = \"|seqstat = \"%(installdir)s/|' "
+                   "%(installdir)s/inparanoid.pl && "
+                   "perl -i -pe 's|blastParser = \"|blastParser = \"%(installdir)s/|' "
+                   "%(installdir)s/inparanoid.pl && "
+                   "perl -i -pe 's|diamondParser = \"|diamondParser = \"%(installdir)s/|' "
+                   "%(installdir)s/inparanoid.pl && "
+                   "perl -i -pe 's|copy\(\"\./|copy\(\"%(installdir)s/|' "
+                   "%(installdir)s/inparanoid.pl && "
+                   "perl -i -pe 's|\./\$blastParser|\$blastParser|g' "
+                   "%(installdir)s/inparanoid.pl && "
+                   "perl -i -pe 's|#!.*perl|#!/usr/bin/env perl|' %(installdir)s/*.pl"]
```

Comment:
It probably makes sense to make these separate accounts, since that way we can more easily tell which command fails (if that happens).
Also, we typically use `sed -i` rather than `perl -i -pe`:

```python
postinstallcmds = [
    "rm -rf %(installdir)s/docker/ %(installdir)s/singularity/",
    "chmod a+rx %(installdir)s/*.pl",
    "chmod -x %(installdir)s/*.jar",
    "sed -i 's|helpfile = \"help\"|helpfile = \"%(installdir)s/help\"|' %(installdir)s/inparanoid.pl",
    ...,
]

Reply:
can also use `fix_perl_shebang_for = ['*.pl']` instead of the `perl -i -pe 's|#!.*perl|#!/usr/bin/env perl|' %(installdir)s/*.pl` command

Reply:
`perl` is a requirement, that's why I used it instead of `sed`. I can change for `sed` if you want.

How to do separate accounts in the postinstallcmds ?

Reply:
`postinstallcmds` is a list of strings; see boegel's example above